hyppo/kgof/_utils.py

```diff
@@ -0,0 +1,310 @@
+"""A module containing convenient methods"""
+from __future__ import print_function
+from __future__ import division
+from __future__ import unicode_literals
+from __future__ import absolute_import
+
+from builtins import zip
+from builtins import int
+from builtins import range
+from future import standard_library
+standard_library.install_aliases()
+from past.utils import old_div
+from builtins import object
+
+import autograd.numpy as np
+import time
+
+class ContextTimer(object):
```

Comment:
I was under the impression that timeit runs the code over and over again to get an average execution time which would slow down module response time for the user, not sure if I'm off-base. The way ContextTimer works is it just times the beginning of the block of code to the end for that one execution.

Reply:
you can set `timeit` to run once. Running it multiple times with `timeit` averaging is better anyway. Either way, not sure how timing any code helps with calculating the test statistic

Reply:
I see what you mean, wrote that as an attempt to optimize test locations and the Gaussian kernel width by maximizing a test power criterion. Although this is only necessary / useful when handling isotropic Gaussian kernel. I can honestly remove that one entirely given it's a niche use case if you think that would be better for this particular merge and work more on adding additional kernels next semester?

Reply:
I think that's what I'll do -- seems like it'll be a more realistic goal for the next week because I know the FSSD test works well on its own. Hope that's ok.

Reply:
Removed.